### ISSUE TITLE
Minor training config updates, in prep for large-scale hyperparam tuning

### ISSUE THREAD

**Issue by jiwoncpark** [jiwoncpark/node-to-joy#14]
- [x] Make config file
- [x] Don't make validation batch size same as training batch size
- [x] Don't make validation batch size same as validation set size
- [x] Make number of parallel workers configurable from training script
- [x] `InferenceManager` should evaluate precision, accuracy, log posterior at the truth
- [x] `InferenceManager` should plot calibration

**Comment by jiwoncpark** [comment]
@mueland Please add to the list as you see fit!
